Open_Science_New.html

```diff
@@ -376,56 +428,44 @@ <h2>Privacy Concerns</h2>
 		  </td>
 	      </tr></table>
 	    </section>
-	  </section>
-	  
-	  <!-- Brainhack -->
+
 	  <section data-transition="fade">
 	    <table class="reveal">
-	      <h2><a href="">Hackathons</a></h2>
 	      <tr>
 		<td width="75%" style="vertical-align: middle">
-		  <img src="images/C-BASE.png">
+		  <a href="http://loris.ca/LORIS_Governance_1904.pdf"><img src="images/LORISgov.png"></a>
 		</td>
 		<td width="25%" style="vertical-align: middle">
-		  <p> Important value of hackathons to data sharing initiatives</p>
-		  <img src="images/Brainhack.png">
+		  <p>See how MCIN adheres to Open Science ethics.<br>
```

Comment:
- [ ] put a smaller image of the whole of page 1 of the doc. 
- [ ] in the right column, try : 
> Released in 2019: 
> LORIS best practices and operational procedures for data privacy and security

Reply:
Also could you remind me -- does this need to go in this column? I think it should be "filed" under the LORIS column. (i do see it's a good segue tho)